person.rb

```diff
@@ -25,4 +26,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def rent_person(rent)
+    rentals << rent
+  end
```

Comment:
- [ ] In your implementation, the association is not being created since all you are doing is pushing the rental into the `@rentals` array. In order for this association to be created we need to create a new rental by calling the rental class. Example:

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```